aip/general/0136.md

```diff
@@ -44,6 +44,11 @@ services. The bullets below apply in all three cases.
   - The name **must not** contain prepositions ("for", "with", etc.).
   - The verb in the name **should not** contain any of the standard method verbs ([Get][],
     [List][], [Create][], [Update][], [Delete][]).
+  - The name **must not** include the term `Async`. Instead, if the intention is
+    to differentiate between immediate and long-running RPCs, the verb `Begin` **may**
+    be used for this purpose. For example, to create a long-running book creation RPC
+    (if the standard `CreateBook` method was designed before long-running aspects were
```

Comment:
I think "longrunning" was discussed as the suffix offline? would that not work?

1. it's a suffix: easy to add to the end and still have pattern matching on the relevant prefix portion
2. hints at the major difference: LRO vs non-LRO.

Reply:
This is the only blocker for my approval, so if you're happy to change it I'd love that!

Reply:
If you can continue discussion in the internal space to get consensus, I'll fix it up tomorrow UK time.

Reply:
ack, we have the design meeting tomorrow too, where we could discuss.

Reply:
Yup. I probably won't be able to make the meeting tomorrow, but I'm happy with whatever everyone else agrees :)

Reply:
Suffix, `LongRunning`, works for me.

Reply:
That sounds like a quorum. Thanks - will update tomorrow UK time, and ping for final approvals.